Thumbnail mgmt issue

Trying to run with `--subject=phys`

```
INFO                          13: Theory of Groups (HTML5AppNode): 1 file
INFO                          14: Nuclear Physics (HTML5AppNode): 1 file
INFO                          15: Quantum Field Theory and Particle Physics (HTML5AppNode): 1 file
INFO                          16: Astrophysics (HTML5AppNode): 1 file
INFO                       Back Matter (TopicNode): 2 descendants
INFO                          Index (HTML5AppNode): 1 file
INFO                          Glossary (HTML5AppNode): 1 file
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/ricecooker/classes/nodes.py", line 459, in validate
    return super(TopicNode, self).validate()
  File "/usr/local/lib/python3.8/site-packages/ricecooker/classes/nodes.py", line 428, in validate
    return super(TreeNode, self).validate()
  File "/usr/local/lib/python3.8/site-packages/ricecooker/classes/nodes.py", line 240, in validate
    f.validate()
  File "/usr/local/lib/python3.8/site-packages/ricecooker/classes/files.py", line 411, in validate
    assert ext in self.allowed_formats, "{} must have one of the " \
AssertionError: ThumbnailFile must have one of the following extensions: ['jpg', 'jpeg', 'png'] (instead, got 'none' from 'chefdata/phys/thumbnails/16: Oscillatory Motion and Waves.None')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./sushichef.py", line 1426, in <module>
    chef.main()
  File "/usr/local/lib/python3.8/site-packages/ricecooker/chefs.py", line 289, in main
    self.run(args, options)
  File "/usr/local/lib/python3.8/site-packages/ricecooker/chefs.py", line 280, in run
    uploadchannel_wrapper(self, args, options)
  File "/usr/local/lib/python3.8/site-packages/ricecooker/commands.py", line 30, in uploadchannel_wrapper
    uploadchannel(chef, **args_and_options)
  File "/usr/local/lib/python3.8/site-packages/ricecooker/commands.py", line 103, in uploadchannel
    config.PROGRESS_MANAGER.set_tree(create_initial_tree(channel))
  File "/usr/local/lib/python3.8/site-packages/ricecooker/commands.py", line 227, in create_initial_tree
    tree.validate()
  File "/usr/local/lib/python3.8/site-packages/ricecooker/managers/tree.py", line 24, in validate
    return self.channel.validate_tree()
  File "/usr/local/lib/python3.8/site-packages/ricecooker/classes/nodes.py", line 223, in validate_tree
    assert child.validate_tree()
  File "/usr/local/lib/python3.8/site-packages/ricecooker/classes/nodes.py", line 223, in validate_tree
    assert child.validate_tree()
  File "/usr/local/lib/python3.8/site-packages/ricecooker/classes/nodes.py", line 223, in validate_tree
    assert child.validate_tree()
  [Previous line repeated 1 more time]
  File "/usr/local/lib/python3.8/site-packages/ricecooker/classes/nodes.py", line 221, in validate_tree
    self.validate()
  File "/usr/local/lib/python3.8/site-packages/ricecooker/classes/nodes.py", line 461, in validate
    raise InvalidNodeException("Invalid node ({}): {} - {}".format(ae.args[0], self.title, self.__dict__))
ricecooker.exceptions.InvalidNodeException: Invalid node (ThumbnailFile must have one of the following extensions: ['jpg', 'jpeg', 'png'] (instead, got 'none' from 'chefdata/phys/thumbnails/16: Oscillatory Motion and Waves.None')): 16: Oscillatory Motion and Waves - {'source_id': 'https://phys.libretexts.org/Bookshelves/College_Physics/Book%3A_College_Physics_(OpenStax)/16%3A_Oscillatory_Motion_and_Waves', 'author': 'OpenStax', 'aggregator': '', 'provider': '', 'tags': [], 'domain_ns': None, 'questions': [], 'extra_fields': {}, 'files': [<ricecooker.classes.files.ThumbnailFile object at 0x7f95f49cfd00>], 'children': [], 'descendants': [], 'parent': <ricecooker.classes.nodes.TopicNode object at 0x7f95f45fae80>, 'node_id': None, 'content_id': None, 'title': '16: Oscillatory Motion and Waves', 'language': 'en', 'description': '', 'derive_thumbnail': False, 'thumbnail': <ricecooker.classes.files.ThumbnailFile object at 0x7f95f49cfd00>}
INFO     Deleting chef temp files at /app/.ricecooker-temp
```

--- a/sushichef.py
+++ b/sushichef.py
@@ -354,7 +354,7 @@ def save_thumbnail(url, title):
     else:
         img_buffer = BytesIO(r.content)
         img_ext = imghdr.what(img_buffer)
-        if img_ext != "gif":
+        if img_ext in ["jpeg", "png"]:
             filename = "{}.{}".format(title, img_ext)
             base_dir = build_path([DATA_DIR, DATA_DIR_SUBJECT, "thumbnails"])
             filepath = os.path.join(base_dir, filename)

